compiledb-gen-parser

```diff
@@ -70,8 +70,8 @@ def parse_flags(build_log, proj_dir, inc_prefix, verbose):
     cpp_compile_regex = re.compile("(.*-?[gc]\+\+ )|(.*-?clang\+\+ )")
 
     # Leverage make --print-directory option
-    make_enter_dir=re.compile("make\[\d+\]: Entering directory [`\'\"](?P<dir>.*)[`\'\"]$")
-    make_leave_dir=re.compile("make\[\d+\]: Leaving directory .*$")
+    make_enter_dir=re.compile("[\t ]*make\[\d+\]: Entering directory [`\'\"](?P<dir>.*)[`\'\"][\t ]*$")
+    make_leave_dir=re.compile("[\t ]*make\[\d+\]: Leaving directory .*$")
```

Comment:
Can we benmore flexible here? Use `^\s*make...` intead.

Reply:
Yes, \s is the better one.